### ISSUE TITLE
IAM Identity Center - External IDentity Provider configuration support

### ISSUE THREAD

**Issue by dsultanaabc** [awslabs/landing-zone-accelerator-on-aws#322]
**Is your feature request related to a problem? Please describe.**
Are we able to configure IAM Identity Centre with an External Identity Provider?

We are starting our Control Tower journey. We would like to configure IAM Identity Center with Azure (External Identity Provider) as the source for users with SCIM configured. I'm wondering if there's currently a method of configuring this using LZA?

Or am I expected to deploy this using Custom Configuration via a CFN stack?

**Describe the feature you'd like**
 We would like to configure IAM Identity Center with Azure (External Identity Provider) as the source for users using LZA.

**Additional context**

**Comment by SennaSemakula** [comment]
Hi @dsultanaabc wouldn't you be able to add a provider for Azure in `iam-config.yaml` similar to [sample config](https://github.com/awslabs/landing-zone-accelerator-on-aws/blob/main/reference/sample-configurations/lza-sample-config-cn/iam-config.yaml#L8)? I haven't tried it myself so please let me know if you have any success.

Here is the specification: https://awslabs.github.io/landing-zone-accelerator-on-aws/classes/_aws_accelerator_config.IamConfig.html#providers

**Comment by supitysup** [comment]
Hey I believe that configures SAML in IAM Identity Provider 
We want to configure an IAM Center Centre External Provider, eg Azure

I also haven’t tested so could be wrong!

**Comment by shahbour** [comment]
i don't think it is implemented yet , https://awslabs.github.io/landing-zone-accelerator-on-aws/classes/_aws_accelerator_config.IdentityCenterConfig.html . currently we can just create permissionsets and assign it
